validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java

```diff
@@ -212,7 +214,7 @@ public void onAttestationEvent(BroadcastAttestationEvent event) throws IllegalAr
       // Save headState to check for slashings
       //      this.headState = headState;
     } catch (IllegalArgumentException e) {
-      STATUS_LOG.log(Level.WARN, "Can not produce attestations or create a block" + e.toString());
+      LOG.warn("Can not produce attestations or create a block" + e.toString());
```

Comment:
Probably should be noted on stdout.  The user will lose money as a result of failing to perform their validator duties.

Reply:
Would an error message output also be warranted for the other two flows that produce `IllegalArgumentException`?

Reply:
Not sure what other two flows you mean.  The basic rule I'm applying is that if it's going to cost me money, I want it to go to STDOUT.  Failing create attestations and blocks will do that.  Failing to process them may not (it may actually be an invalid block), but it's a bit border line because if there's a bug that caused a consensus failure then I'll drop out of sync and no longer be attesting to the right chain (and thus stop being rewarded).

Reply:
Sorry, I meant that for the above comment (on `KeystoresValidatorKeyProvider.java`), as that has the two `IllegalArgumentException` lines prior to the log case. 

All three flows result in the same outcome, namely a runtime subclass.

Reply:
Applying the principle of reporting things that would cause you to lose money to stdout, then yes anything that causes us to not load validator keys should be reported.  Probably should cause startup to fail entirely actually - not quite sure where the scope boundary for this work is in that mix though. :)

Reply:
Once you begin pulling at a thread ...I suppose 🙄 

When a `RuntimeException` is thrown during those flows (or any time during startup, it seems), Teku stops loading with the `Exception` logged (to the log file, not the console).

Reply:
`BeaconNode.start()` is the line that I'll change to console logging, as that'll catch the validator password keyfile too